Gems/LmbrCentral/Code/Source/Shape/EditorAxisAlignedBoxShapeComponentMode.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzToolsFramework/ComponentModes/BoxComponentMode.h>
+
+namespace LmbrCentral
+{
+    class EditorAxisAlignedBoxShapeComponentMode : public AzToolsFramework::BoxComponentMode
+    {
+    public:
+        AZ_CLASS_ALLOCATOR_DECL
+
+        EditorAxisAlignedBoxShapeComponentMode(const AZ::EntityComponentIdPair& entityComponentIdPair, AZ::Uuid componentType);
+
+        AZStd::string GetComponentModeName() const override;
```

Comment:
suggestion: We quite often include a comment for the overrides to group them together, so here we'd have...

```c++
// AzToolsFramework::BoxComponentMode overrides ...
AZStd::string GetComponentModeName() const override;
```

Reply:
Done :) Thanks for the infos !

Reply:
No problem! I'll kick off a build now 👍